<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TalpaMainWindow</class>
 <widget class="QMainWindow" name="TalpaMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>722</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Talpa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_model"/>
    <addaction name="actionLoad_model"/>
    <addaction name="separator"/>
    <addaction name="actionImport_kite_Scene"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLog"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout_Talpa"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindows"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionHelp">
   <property name="text">
    <string>Online Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Logger</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAbout_Talpa">
   <property name="text">
    <string>About Talpa</string>
   </property>
  </action>
  <action name="actionSave_model">
   <property name="text">
    <string>Save model</string>
   </property>
  </action>
  <action name="actionLoad_model">
   <property name="text">
    <string>Load model</string>
   </property>
  </action>
  <action name="actionImport_kite_Scene">
   <property name="text">
    <string>Import kite.Scene</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
